sumologic/sumologic_data_forwarding_rule.go

```diff
@@ -62,7 +62,7 @@ func (s *Client) DeleteDataForwardingRule(indexId string) error {
 type DataForwardingRule struct {
 	IndexId       string `json:"indexId"`
 	DestinationId string `json:"destinationId"`
-	Enabled       bool   `json:"enabled,omitempty"`
+	Enabled       bool   `json:"enabled"`
```

Comment:
Why are we making this change?  This will send value as `false` instead of not serialising it in json response(which eventually gets treated as `null`)

Reply:
The bad part is that `enabled = false` is treated as null.... and whenever we are trying to set `false`, it is not sent in the API call... 

We are not setting any default for `enabled` .. so it will not be send as `false` in case left blank (see [here](https://github.com/SumoLogic/terraform-provider-sumologic/pull/691/files/138fe92703280402b4d5c5383b44671a5b1bc8ef#diff-a6863bdbf853ea5b139b3ca10ea5fe15699db83c96727384162c8fe87f5385f3L36)).

I've tested this scenario..